pipelines/cdp/cdp_pipeline/customer_data_platform.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import apache_beam as beam
+from apache_beam import Pipeline, PCollection
+from apache_beam.options.pipeline_options import PipelineOptions
+from apache_beam.io.gcp.bigquery import ReadFromBigQuery, WriteToBigQuery
+import json
+from apache_beam.transforms.window import FixedWindows
+from apache_beam.transforms.trigger import AfterWatermark, AfterProcessingTime, AccumulationMode
+from cdp_pipeline.options import MyPipelineOptions
+
+# Get the schema of bigquery output table
+with open('./schema/unified_table.json') as schema_file:
+        output_schema = json.load(schema_file)
+        
+def left_join(key_value_pair):
+    logging.info(key_value_pair)
+    key, values = key_value_pair
+    trans_values, coupon_redempt_values = values
+    logging.info(trans_values)
+    logging.info(coupon_redempt_values)
```

Comment:
This is probably too much logging in Dataflow, this will log 3 lines for every processed value.